src/main/java/gregtech/loaders/recipe/chemistry/PolymerRecipes.java

```diff
@@ -369,25 +369,18 @@ private static void polybenzimidazoleProcess() {
         LARGE_CHEMICAL_RECIPES.recipeBuilder().EUt(VA[IV]).duration(100)
                 .fluidInputs(Dichlorobenzidine.getFluid(1000))
                 .fluidInputs(Ammonia.getFluid(2000))
-                .notConsumable(dust, Zinc)
+                .notConsumable(dust, Copper)
```

Comment:
Copper is intentionally a consumed resource for PBI.

This recipe needs to have a tiny dust of copper consumed, and a 9x multiplied recipe consuming a full dust in order to maintain the same game balance as before.

Reply:
Fixed in latest commit.